### DISCUSSION TITLE
Import Vue Carousel 3D

### DISCUSSION THREAD

**Discussion by HarleyBerglund** [quasarframework/quasar#11216] [General - Components / Directives / etc] [answered]
I want to use the carousel from https://wlada.github.io/vue-carousel-3d/ in my Quasar project. But I do not know how i do it. I know how to do it in ordinary Vue but not in Quasar. Can someone help me?

**Comment by pdanpdan** [comment, marked as answer]
The same way, but it looks like it is for vue2, so you can use it in Quasar v1

**Reply by HarleyBerglund** [reply]
Okey, im using Vue3. So I guess i will try to find something simmilar to this for vue 3

**Reply by HarleyBerglund** [reply]
Do you know any good-looking components for that?
@pdanpdan
